plugins/plugin-chart-echarts/src/BoxPlot/transformProps.ts

```diff
@@ -33,11 +35,13 @@ import { defaultGrid, defaultTooltip, defaultYAxis } from '../defaults';
 export default function transformProps(chartProps: ChartProps): EchartsProps {
   const { width, height, formData, queriesData } = chartProps;
   const data: DataRecord[] = queriesData[0].data || [];
+  const coltypes: Array<GenericDataType> = queriesData[0].coltypes || [];
```

Comment:
nit: could be written slightly more compactly as `coltypes: GenericDataType[] = ...`